tests/integration/rdb.tcl

```diff
@@ -283,12 +287,19 @@ start_server {overrides {save ""}} {
             }
 
             # trigger copy-on-write
-            r setrange key$iteration 0 [string repeat B $size]
+            set modified_keys 10
+            for {set k 0} {$k < $modified_keys} {incr k} {
+                r setrange key$key_idx 0 [string repeat B $size]
+                incr key_idx 1
+            }
 
+            # we expects for only 50% change in the COW size, since we are using small keys size (less than a page), and
+            # some keys may fall into the same page and will not create copy-on-write
```

Comment:
i'm not sure the fact that two keys share the same page matters here (i think the comment may need to change, or maybe we should switch to 90%).
if you change one key, you'll get 4kb COW, then possibly the next 8 keys aren't gonna make a difference, but on average after 100 keys you'll get 50kb COW.

maybe we change to 16 keys in each step, and 80%?
```suggestion
            # we expects for at least 50% change in the COW size, since we are using small keys size (less than a page), and
            # some keys may fall into the same page and will not create copy-on-write
```